build.sbt

```diff
@@ -182,8 +184,10 @@ lazy val `material-ui` =
         "@types/recharts" -> "1.8.10",
         "@types/classnames" -> "2.2.10",
         "react-router-dom" -> "5.1.2",
-        "@types/react-router-dom" -> "5.1.2"
-      )
+        "@types/react-router-dom" -> "5.1.2",
+        "react-proxy" -> "1.1.8"
```

Comment:
what is `react-proxy` for? is it related to `hot`?

Reply:
Yes, otherwise, compilation fails